src/main/java/com/ldtteam/structurize/client/BlueprintRenderer.java

```diff
@@ -143,6 +117,11 @@ private void init(final Blueprint blueprint, final Map<Object, Exception> suppre
         blockAccess.setBlockEntities(tileEntitiesMap);
         blockAccess.setEntities(entities);
 
+        if (previewData.getSolidSubstitutionOverride() != null)
+        {
+            blockAccess.setSolidSubstitutionOverride(previewData.getSolidSubstitutionOverride());
```

Comment:
not bad, could we unset this after in the end of the init method?

Reply:
oh right for the other previewdata's which may have not set it